### ISSUE TITLE
OpenGL bounds relaxation

### ISSUE THREAD

**Issue by ulysses4ever** [haskell-infra/hackage-trustees#335]
It's been >half of year since the ticket to bump a couple of upper bounds in OpenGL was opened: https://github.com/haskell-opengl/OpenGL/issues/89

I pinged the maintainers a week ago, but had no reply. Would it be  possible to at least bump `bytestring` bound from `<.011` to `<0.12`? This'd allow building with GHC 9.2 all right.

**Comment by ulysses4ever** [comment]
@andreasabel somehow, OpenGLRaw is already bumped to bytestring<0.12 (l learned it by following your link to Hackage). I did check that OpenGL builds with the bump and _no_ allow-newer.

**Comment by ulysses4ever** [comment]
Yeah, I didn’t check 9.4 as I didn’t have it readily available. I’d be happy if you can fix it for 9.4 too.

**Comment by andreasabel** [comment]
Fixed, see https://github.com/haskell-opengl/OpenGL/issues/89#issuecomment-1142253949
The GHC 9.4 bump needed for `OpenGLRaw` should be done separately, after alerting the maintainer.
